.ci/Jenkinsfile

```diff
@@ -16,6 +16,7 @@ pipeline {
     JOB_GCS_BUCKET = credentials('gcs-bucket')
     PIPELINE_LOG_LEVEL='INFO'
     GIT_REFERENCE_REPO = '/var/lib/jenkins/.git-references/apm-integration-testing.git'
+    ELASTIC_STACK_VERSION = "${ params?.ELASTIC_STACK_VERSION?.trim() ? params.ELASTIC_STACK_VERSION.trim() : stackVersions.edge() }"
```

Comment:
`stackVersions.edge()`, will it work? IIRC, if it was a step then it requires the `script` closure, just in case

Reply:
Interpolations are scripts, this pipeline works as expected

```
pipeline {
    agent { label 'master'}
    environment {
        FOO = "${stackVersions.release()}"
    }
    stages {
        stage('Hello') {
            steps {
                echo "Hello World ${FOO}"
            }
        }
    }
}
```